wBoooksApi/db/migrate/20180611203152_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[5.2]
```

Comment:
You'll need to remove this migration too. After that you will need to do rails db:drop && rm db/schema.rb && rails db:create rails db:migrate